### DISCUSSION TITLE
Ask the community for help

### DISCUSSION THREAD

**Discussion by 305916500** [ClassicOldSong/moonlight-android#6] [Q&A] [answered]
When I'm using the multi-touch or Absolute-touch mode on my Pad, if I long-press with my finger, it triggers the right-click function. How can I turn off this long-press-to-right-click feature?

**Comment by ClassicOldSong** [comment]
With multi touch, long press for right click is defined by Windows, thus there's no way to disable.

With absolute touch the behavior is inherited from the og moonlight, currently there is no way to disable it.

**Reply by ClassicOldSong** [reply, marked as answer]
With a quick search I found this tutorial for disabling press to right click on Windows: https://www.elevenforum.com/t/enable-or-disable-press-and-hold-for-right-clicking-in-windows-11.6330/

**Reply by 305916500** [reply]
Thank you very much for your explanation , I understand the principle now.
